lib/pages/chat/seen_by_row.dart

```diff
@@ -1,22 +1,25 @@
 import 'package:fluffychat/pages/chat/events/message_time_style.dart';
-import 'package:fluffychat/resource/image_paths.dart';
+import 'package:fluffychat/widgets/matrix.dart';
 import 'package:flutter/material.dart';
 
 import 'package:fluffychat/pages/chat/chat.dart';
 import 'package:fluffychat/utils/room_status_extension.dart';
-import 'package:flutter_svg/flutter_svg.dart';
 import 'package:matrix/matrix.dart';
 
+enum MessageStatus { sending, sent, hasBeenSeenByOne, hasBeenSeenByAll }
+
 class SeenByRow extends StatelessWidget {
   final ChatController controller;
```

Comment:
IMO: should not use `controller` in `StatelessWidget`. You can refactor it

Reply:
It's a new rule, I just remembered it for u

Reply:
@sherlockvn @nqhhdev https://github.com/linagora/twake-on-matrix/pull/477/commits/63403a93d0b044ee01ea015f6bd1413892ac1565 is that ok ?